src/primitives/ray.jl

```diff
@@ -42,3 +42,11 @@ origin(r::Ray) = r.p
 The direction of the ray.
 """
 direction(r::Ray) = r.v
+
+
+
+function Base.in(p::Point{Dim, T}, r::Ray{Dim, T}) where {Dim,T}
+	return p ∈ Line(r(zero(T)), r(one(T))) && sum((p - r.p) .* r.v) >= 0 # additional check
+end
```

Comment:
Incorrect indentation and tons of unnecessary whitespace. We don't use `return` either. Appreciate if you can comply to the code style of the project.